apps/container/src/components/UserMenu/UserMenu.test.ts

```diff
@@ -0,0 +1,9 @@
+import { describe, expect, test } from 'vitest';
+
+import UserMenu from './UserMenu.vue';
+
+describe('<UserMenu />', () => {
+  test('renders component', () => {
+    expect(UserMenu).toBeTruthy();
+  });
+});
```

Comment:
seria legal realmente criar testes que validem que o componente continua tendo seus comportamentos preservados quando ele for alterado